client/src/routes/automation/BlocklyComponent.svelte

```diff
@@ -27,7 +27,8 @@
 </script>
 
 <p>
-    <button on:click={blocklyService.convertToBlock()}>Convert the blocks !</button>
+    <button on:click={blocklyService.convertToBlock()}>TEST: Convert the blocks
```

Comment:
```suggestion
    <button on:click={blocklyService.convertToBlock}>TEST: Convert the blocks
```
Faut donner une reference de fonction, là ici ta fonction convertToBlock se lance dès que la page se charge  :^)

Reply:
Done